CHANGELOG.md

```diff
@@ -42,8 +51,7 @@ The format is based on [Keep a Changelog], and this project adheres to
 ### Changed
 
 - Upgraded Python to version 3.11 to take advantage of the addition of
-  [fine-grained error locations in tracebacks](https://docs.python.org/3/whatsnew/3.11.html#whatsnew311-pep657)
-  to help with debugging errors.
+  [fine-grained error locations in tracebacks] to help with debugging errors.
```

Comment:
Accidentally dropped the link?

Reply:
I moved it to the bottom of the file.

Reply:
but you still have brackets on this line?

Reply:
yes, that's how it works: I can use `[X]` within the body of the doc, then put the following at the bottom of the doc so that it does not clutter the main text:

```
[X]:
    https://relevant-url
```

Reply:
See https://riptutorial.com/markdown/example/2215/reference-link for the various forms